Justfile

```diff
@@ -0,0 +1,21 @@
+[private]
+default:
+    @just --list
+  
+# Format all code
+format:
+    cargo +nightly fmt
+
```

Comment:
A clippy alias matching CI would also be helpful:

`cargo +stable clippy --all-features --workspace --all-targets`